diffsims/tests/test_library/test_structure_library.py

```diff
@@ -21,6 +21,17 @@
 
 from diffsims.libraries.structure_library import StructureLibrary
 
+def test_get_library_size():
+    identifiers = ['a', 'b']
+    structures = [1, 2]
+    rotation_list = [[3,1], [4,5]]
+    orientations = [1, 2]
```

Comment:
This wouldn't be a legitimate input. Has to be a list of lists of tuples really.

Reply:
@norskie do you understand what the issue is here?

Reply:
Yes! [(3,1), (4,5)] is the correct input for rotation_list, if I am not mistaken. I will attempt to do fix the requested changes today :)

Reply:
See my review comment above. Each phase has an associated list of tuples. So you're looking at something like `[[(0,0,0),(0,1,0)],[(0,1,0),(0,2,0)]]`

Reply:
Update: Changed rotation_list to list of tuples

Reply:
I see, thank you.